metrics.sh

```diff
@@ -25,3 +25,13 @@ for service in 'compute' 'network' 'volume'; do
 		echo "${metric}{cloud=\"${OS_CLOUD}\",project=\"${os_project}\",type=\"limit\"} $limit"
 	done
 done
+
+
+metric=openstack_server
```

Comment:
Should probably be `openstack_server_status`

Reply:
It was in my first iteration. But then I thought: shall we reflect the attributes in the metric? That's really the number of existing servers. Shall the metric name reflect the fact that the list is broken down by status?

Reply:
Ok, I'm sold.

/lgtm